frontend/src/i18n/locales/en.yaml

```diff
@@ -345,7 +345,7 @@ manage:
     unlisted:
       note: This video is unlisted.
       explanation: >
-        Unlisted videos can only be found by sharing its direct link. This is equivalent to a password protection.
+        Unlisted videos can only be found by sharing its direct link or via its series.
```

Comment:
Hm I'm actually unsure if this is correct gramatically since `unlisted videos` is plural and `its` is a singular pronoun. I'd probably change this to either start with `An unlisted video can...` or replacing the `its` with `their`. Adapt accordingly in the german translation, though somehow the german one reads more gramatically correct to me lol.